frontend/src/routes.tsx

```diff
@@ -102,7 +101,7 @@ const routes = [
       },
       {
         path: "settings",
-        element: <SettingsLayout />,
```

Comment:
Can you see anything wrong in the UI by doing this?

Reply:
should be fixed now